nestkernel/mpi_manager_impl.h

```diff
@@ -57,10 +54,9 @@ extern MPI_Comm comm;
 /* ------------------------------------------------------
    The following datatypes are defined here in communicator_impl.h
    file instead of as static class members, to avoid inclusion
-   of mpi.h in the .h file. This is necessary, because on
-   BlueGene/L mpi.h MUST be included FIRST. Having mpi.h in
-   the .h file would lead to requirements on include-order
-   throughout the NEST code base and is not acceptable.
+   of mpi.h in the .h file. Having mpi.h in the .h file would
+   lead to requirements on include-order throughout the NEST
+   code base and is not acceptable.
```

Comment:
```suggestion
   of mpi.h in the .h file, which caused problems on BlueGene systems. 
   Having mpi.h in the .h file would lead to requirements on include-order 
   throughout the NEST code base and is not acceptable.
```